chipy_org/apps/job_board/forms.py

```diff
@@ -21,12 +21,17 @@ class Meta:
             "location",
             "description",
             "is_sponsor",
+            "affiliation",
             "can_host_meeting",
             "company_website",
             "how_to_apply",
             "agree_to_terms",
         ]
 
+        help_texts = {
+            "affiliation": ("Let us know if this job posting is associated with a third party."),
```

Comment:
### Currently, this is what the affiliation field dropdown looks like when you go to create a job post:
<img width="812" alt="Screen Shot 2021-05-02 at 11 26 25 AM" src="https://user-images.githubusercontent.com/53249319/116820905-03339900-ab3d-11eb-90c8-12cc226e2f68.png">

### I would like for it to look like this instead:
<img width="710" alt="Screen Shot 2021-05-02 at 12 02 43 PM" src="https://user-images.githubusercontent.com/53249319/116821197-82759c80-ab3e-11eb-92c4-f59221e615ba.png">

To achieve this, please make 2 changes.

STEP 1:
In `job_board/forms.py`, line 32, change the text to: 
"Is this job post affiliated with a 3rd party? Please select:"

STEP 2:
In `job_board/templates/job_post_form.html`, within the `<table> </table>` tags, and right before the `else` clause, insert this code.
```
{% elif field == job_post_form.affiliation %}
                <tr>
                    <td colspan="2">
                        {{field.help_text}} {{field}}
                        {{ field.errors }}
                    </td>
                </tr> 
```

Reply:
On second thought, it's possible to add the verbose name on the affiliation field on the job post model and that will take care of affiliation texts showing up without having to change the template.